test-clients/src/main/java/com/hedera/services/bdd/suites/perf/MixedOpsLoadTest.java

```diff
@@ -0,0 +1,285 @@
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SCHEDULE_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TOKEN_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TOPIC_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PAYER_ACCOUNT_NOT_FOUND;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_NOT_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOPIC_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNKNOWN;
+
+public class MixedOpsLoadTest extends LoadTest {
+	private static final Logger log = LogManager.getLogger(MixedOpsLoadTest.class);
+	private static final int NUM_SUBMISSIONS = 100;
+	private final ResponseCodeEnum[] permissiblePrechecks = new ResponseCodeEnum[] {
+			BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED, UNKNOWN
+	};
+
+	private static String sender = "sender";
+	private static String receiver = "receiver";
+	private static String submitKey = "submitKey";
+	private static String topic = "topic";
+	private static String token = "token";
+	private static String schedule = "schedule";
+	private static int messageSize = 1024;
+
+	public static void main(String... args) {
+		parseArgs(args);
+		MixedOpsLoadTest suite = new MixedOpsLoadTest();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runMixedOps()
+		);
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+	protected HapiApiSpec runMixedOps() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		Random r = new Random();
+		AtomicInteger tokenId = new AtomicInteger(0);
+		AtomicInteger scheduleId = new AtomicInteger(0);
+
+		Supplier<HapiSpecOperation[]> mixedOpsBurst = () -> new HapiSpecOperation[] {
+				cryptoTransfer(
+						tinyBarsFromTo(sender, receiver, 1L))
+						.noLogging()
+						.payingWith(sender)
+						.signedBy(GENESIS)
+						.suppressStats(true)
+						.fee(ONE_HBAR)
+						.hasKnownStatusFrom(SUCCESS,
+								OK,
+								INSUFFICIENT_PAYER_BALANCE,
+								UNKNOWN,
+								TRANSACTION_EXPIRED)
+						.hasRetryPrecheckFrom(BUSY,
+								DUPLICATE_TRANSACTION,
```

Comment:
DUPLICATE_TRANSACTION should not be in retry check.

Reply:
@ljianghedera  Thanks ! Same here. I took the retry precheks from other LoadTests like `SubmitMessageLoadTest`. If we are modifying we need to modify in every place and make sure rest works fine. What do you think?

Reply:
I see your point. I would suggest to change only the current file and leave other places alone for now. The reason is if we change anything, we need to run the tests to make sure nothing is broken. Later on when we need to work on the other places, I think we can make the change at that time.

If the DUPLICATION_TRANSACTION is in the "retryprecheck", it won't cause failure, but may reduce the capacity for the client to send higher TPS to server.